packages/hash/frontend/src/components/layout/MainContentWrapper.tsx

```diff
@@ -62,7 +77,158 @@ export const MainContentWrapper: FunctionComponent = ({ children }) => {
           </IconButton>
         </Tooltip>
       </Fade>
-      <Main sidebarOpen={sidebarOpen}>{children}</Main>
+      <Main sidebarOpen={sidebarOpen}>
+        {children}
+        <br />
+        <br />
+        <Switch />
+        <br />
+        <Switch size="small" />
+        <br />
+        <Radio sx={{ mr: 2 }} name="btn" value="a" />
+        <Radio name="btn" value="b" />
+        <br />
+        <Checkbox sx={{ mr: 2 }} />
+        <Checkbox checked />
+        <br />
+        <br />
+        <Stack direction="row" alignItems="center" spacing={2}>
+          <TextField
+            defaultValue="small"
+            placeholder="Search for anything"
+            size="small"
+          />
+          <TextField
+            defaultValue="medium"
+            placeholder="Search for anything"
+            size="medium"
+          />
+          <TextField
+            defaultValue="large"
+            placeholder="Search for anything"
+            size="large"
+          />
+        </Stack>
```

Comment:
These are added to aid testing. Will remove once PR is approved

Reply:
We can create a new route like `/playground` where all these components can be kept. The route can have `getStaticProps` which returns `{ notFound: true }` in production.